pkg/cmd/pulumi/schema.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newSchemaCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "schema",
+		Short: "Analyze package schemas",
+		Long: `Analyze package schemas
+
+Subcommands of this command can be used to analyze Pulumi package schemas. This can be useful to check hand-authored
```

Comment:
If this is the correct doc page for what this tool is about, adding a link can be very handy: https://www.pulumi.com/docs/guides/pulumi-packages/schema/